### DISCUSSION TITLE
Cropping in -T fails

### DISCUSSION THREAD

**Discussion by itismepeter** [neperfepx/neper#261] [Q&A] [answered]
Dear Romain or fellow Neper users,

I am trying to generate a cropped tessellation using specified weights from a file using `-morphooptiini`. In most cases, this is trouble-free, but in several cases the cropping itself fails and aborts the generation.

For example, running `neper -T -n 832 -id 117 -domain "cube(1.2,1.2,1.2)" -morphooptiini "coo:random,weight:file(w17.txt)" -transform "crop(cube(0.1,1.1,0.1,1.1,0.1,1.1)),resetcellid" -o id117` with a file [w17.txt](https://github.com/rquey/neper/files/6790401/w17.txt) yields
```
========================    N   e   p   e   r    =======================
Info   : A software package for polycrystal generation and meshing.
Info   : Version 4.2.0
Info   : Built with: gsl|muparser|opengjk|openmp|nlopt|libscotch (full)
Info   : Running on 1 threads.
Info   : <https://neper.info>
Info   : Copyright (C) 2003-2021, and GNU GPL'd, by Romain Quey.
Info   : No initialization file found (`/home/vookp/.neperrc').
Info   : ---------------------------------------------------------------
Info   : MODULE  -T loaded with arguments:
Info   : [ini file] (none)
Info   : [com line] -n 832 -id 117 -domain cube(1.2,1.2,1.2)
         -morphooptiini coo:random,weight:file(w17.txt) -transform
         crop(cube(0.1,1.1,0.1,1.1,0.1,1.1)),resetcellid -o id117
Info   : ---------------------------------------------------------------
Info   : Reading input data...
Info   : Creating domain...
Info   : Creating tessellation...
Info   :   - Setting seeds...
Info   :   - Generating crystal orientations...
Info   :   - Running tessellation...
Info   : Transforming tessellation...
Info   :   - Cropping...
Aborted
```

Do you have any clue what might be the reason behind this?

Thank you in advance.
Best regards,
Peter

**Comment by rquey** [comment, marked as answer]
Thanks @itismepeter for the bug report.  This is now fixed (version `4.2.1-5`).
